src/impl/tpls/KokkosBlas1_nrm2_tpl_spec_decl.hpp

```diff
@@ -91,8 +91,8 @@ Kokkos::View<const double*, LAYOUT, Kokkos::Device<ExecSpace, MEMSPACE>, \
     if (numElems < static_cast<size_type> (INT_MAX)) { \
       nrm2_print_specialization<RV,XV>(); \
       int N = numElems; \
-      int one = 1; \
-      R() = HostBlas<double>::nrm2(N,X.data(),one); \
+      int int_one = 1; \
```

Comment:
We could use `Kokkos::ArithTraits<int>::one()` here but it's a detail.
On the other hand I see above on line 91: `static_cast<size_type>(INT_MAX)`, this would probably benefit even more of `Kokkos::ArithTraits<size_type>::max()`

Reply:
This is a good point, I decided to hold off and approach more systematically later (there will be more occurrences) when time isn't crunched.

Reply:
Agreed, thanks for making this PR to fix the issues with mkl : )